pkg/fakerp/routes.go

```diff
@@ -19,6 +19,7 @@ func (s *Server) SetupRoutes() {
 	s.router.Put(s.basePath, s.handlePut)
 	s.router.Get(filepath.Join("/admin", s.basePath), s.handleGet)
 	s.router.Put(filepath.Join("/admin", s.basePath), s.handlePut)
+	s.router.Put(filepath.Join("/admin", s.basePath, "/listBackups"), s.handleListBackups)
```

Comment:
This should be changed to a 
```suggestion
	s.router.Get(filepath.Join("/admin", s.basePath, "/listBackups"), s.handleListBackups)
```

Reply:
good catch thanks